flow/flow.go

```diff
@@ -108,14 +108,17 @@ const (
 	DNSLayer ExtraLayers = 2
 	// DHCPv4Layer extra layer
 	DHCPv4Layer ExtraLayers = 4
+	// DNSLayerAll extra layer, intercept every packet
+	DNSLayerAll ExtraLayers = 8
 	// ALLLayer all extra layers
-	ALLLayer ExtraLayers = 255
+	ALLLayer ExtraLayers = 247
 )
 
 var extraLayersMap = map[string]ExtraLayers{
-	"VRRP":   VRRPLayer,
-	"DNS":    DNSLayer,
-	"DHCPv4": DHCPv4Layer,
+	"VRRP":            VRRPLayer,
+	"DNS":             DNSLayer,
+	"DHCPv4":          DHCPv4Layer,
+	"DNS_ALL_PACKETS": DNSLayerAll,
```

Comment:
I would keep only one option DNS, getting all the packets. The only thing we need to check that we can request using gremlin, we should add functional test for that

Reply:
Right now you can request this via the --extra-layer flag. Isn't this enough?

Reply:
I would just keep DNS, I do not see the point to get multiple level of analysis for DNS. My point was since we change the way the DNS fields are reported in flow we need to ensure we can still filter flow based on DNS fields. Adding functionals test will help to ensure it is working.

Reply:
Added one simple test which ensures we can filter flows by certain DNS fields